client/api_handler.py

```diff
@@ -18,6 +32,11 @@ def get_metadata(url, response_type):
 
     sanitized_url = url_utils.get_redirect_url(head)
 
+    data = check_db_for_url(sanitized_url)   # If data from db is None, continue and parse the website
```

Comment:
let's wrap this into a config variable like
USE_DB so the developer can decide whether or not to use db